SitemapElementInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle;
+
+/**
+ * Documents persisted with phpcr-odm should implement that interface.
+ * They also need to create a property with the name "visible_for_sitemap" to be recognized
+ * by the default provider, that generates the UrlInformation object.
```

Comment:
lets say:

Models implementing this interface can indicate whether they should be listed in the sitemap or not.

For PHPCR-ODM documents used with the Doctrine\Phpcr\Sitemap.. , make sure to call the property that stores this information "visible_for_sitemap", as the provider uses the property in the query rather than loading all documents and checking the method.